packages/date/src/index.js

```diff
@@ -156,9 +156,7 @@ export function getLastPeriod( period, compare ) {
 			'week' === period
 				? primaryStart
 						.clone()
-						.subtract( 1, 'years' )
-						.week( primaryStart.week() )
-						.startOf( 'week' )
+						.subtract( 364, 'days' )
```

Comment:
Will this still work okay in leap years?

Reply:
> Will this still work okay in leap years?

Yes because it's an even 52 weeks. On a leap year the dates will be 2 days apart instead of 1 but it will still be the same week number (eg. this week is week 42).